bps/serial.go

```diff
@@ -52,3 +53,19 @@ func (b *BPS) Scan(value interface{}) error {
 func (b *BPS) Value() (driver.Value, error) {
 	return b.String(), nil
 }
+
+// UnmarshalText implements the encoding.TextUnmarshaler interface.
+func (b *BPS) UnmarshalText(text []byte) error {
+	buf := bytes.NewBuffer(text)
+	if buf.Len() == 0 {
+		return errors.New("BPS.UnmarshalText: no data")
+	}
```

Comment:
```suggestion
	v := string(text)
	if v == "" {
		return errors.New("BPS.UnmarshalText: no data")
	}
```

I think it's more simple.